benchmarks/auto_benchmarking.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
perhaps you can add this as a subcommand of the existing `garage` script in `scripts/`?

e.g. so that you could do `garage benchmark foo bar baz`

Reply:
If this is done then adding an example to the README would be great. Maybe garage_benchmarks should have its own README?

Reply:
Ye I haven't started to write any high level docs. I'll definitely add those.

Reply:
@ryanjulian After some thinking I think it's probably better not to put this script in `garage`? My point is the installation for benchmarking is purely optional, so it makes sense to me to create a script under `benchmarks` folder. Though I think I could also make it useful for general benchmarking, which also justifies the point. So something like this:
```
benchmark list
benchmark run b1 b2
...
````

Reply:
yes, we can consider making it part of the `garage` command later.

i recommend you make the script named `garage-benchmark` because who knows what other executable someone has on their machine named "benchmark".

you can add this line to your setup.py to ensure your command is installed in the environment (without having to use ./ or add to PATH): https://github.com/rlworkgroup/garage/blob/master/setup.py#L100

Reply:
[This](https://github.com/rlworkgroup/garage/blob/ecd749fe76be759a9e402b88810eb99a566c261e/benchmarks/setup.py#L22-L25) should do the same magic. You just make sure you're in `/benchmarks`.